src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/QuestionnaireService.kt

```diff
@@ -63,10 +64,14 @@ class QuestionnaireService(
           dateRequired = questionnaireAnswer.dateRequired,
         )
           .apply {
-            if (includeNextQuestion) {
```

Comment:
Had a look at the full response in preprod and there was just too much unnecessary information - so just pulled in minimum for identification an readability